src/gui_common/thrive_theme.tres

```diff
@@ -363,7 +365,7 @@ corner_radius_bottom_left = 10
 expand_margin_top = 20.0
 
 [resource]
-default_font = ExtResource( 8 )
+default_font = ExtResource( 9 )
```

Comment:
Did @Kasterisk approve on changing the *default* font to a different one? I think this may have pretty wide consequences...

Reply:
That may be a mistake.

Reply:
Didn't saw that one... Well, thanks for noticing.